pkg/binding/bindingpolicy-resolver.go

```diff
@@ -235,6 +234,11 @@ func (resolver *bindingPolicyResolver) EnsureObjectData(bindingPolicyKey string,
 			bindingPolicyKey)
 	}
 
+	// Now the resolver's mutex is not held, so the resolution just fetched could be concurrently
```

Comment:
`Now` makes sense to us code maintainers since things are changing, but to external readers it wont.

Reply:
I expect this method's implementation will only be read by code maintainers.